cache/cache.cpp

```diff
@@ -559,6 +559,41 @@ static int dnet_cmd_cache_io_lookup_new(struct cache_manager *cache,
 	return dnet_send_reply(st, cmd, response.data(), response.size(), 0);
 }
 
+static int dnet_cmd_cache_io_remove(struct cache_manager *cache,
+				    struct dnet_cmd *cmd,
+				    struct dnet_io_attr *io,
+				    struct dnet_cmd_stats *cmd_stats) {
+	ioremap::elliptics::dnet_remove_request request{io->flags, {0, 0}};
```

Comment:
Old remove should ignore `DNET_IO_FLAGS_CAS_TIMESTAMP`, so it should remove the key.